services/trace/traceService.ts

```diff
@@ -0,0 +1,96 @@
+import { Project, ProjStatus } from '../../entities/project';
+import { errorMessages } from '../../utils/errorMessages';
+import { ProjectStatus } from '../../entities/projectStatus';
+import { RedisOptions } from 'ioredis';
+// tslint:disable-next-line:no-var-requires
+const Queue = require('bull');
+
+// There is shared redis between giveth.io and trace.giveth.io notify each other about verifiedCampaigns/project update
+const redisConfig: RedisOptions = {
+  host: process.env.SHARED_REDIS_HOST,
+  port: Number(process.env.SHARED_REDIS_PORT),
+};
+if (process.env.SHARED_REDIS_PASSWORD) {
+  redisConfig.password = process.env.SHARED_REDIS_PASSWORD;
+}
+
+const updateCampaignQueue = new Queue('trace-campaign-updated', {
+  redis: redisConfig,
+});
+const updateGivethIoProjectQueue = new Queue('givethio-project-updated', {
+  redis: redisConfig,
+});
+
+export interface UpdateCampaignData {
+  title?: string;
+  campaignId?: string;
+  description?: string;
+  verified?: boolean;
+  archived?: boolean;
+}
+
+export const updateCampaignInTrace = async (
+  project: Project,
+): Promise<void> => {
+  try {
+    if (!project.isTraceable) {
+      console.log(
+        'updateCampaignInTrace(), the project is not a trace campaign',
+        {
+          projectId: project.id,
+        },
+      );
+      return;
+    }
+    const payload: UpdateCampaignData = {
+      campaignId: project.traceCampaignId,
+      title: project.title,
+      description: project.description,
+      verified: project.verified,
```

Comment:
Can verification change here?

Reply:
Yeah, admin may change it in admin bro panel